packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -41,63 +46,75 @@ export function getXMLViewDiagnostics(opts: {
   if (opts.context.manifestDetails.flexEnabled) {
     actualValidators.element.push(validators.validateNonStableId);
   }
-  const externalValidators: UI5ValidatorsConfig<AnnotationIssue> = cloneDeep(
-    externalDefaultValidators
-  );
+  const externalFeValidators: UI5ValidatorsConfig<AnnotationIssue> =
+    cloneDeep(feValidators);
+  const externalBindingValidators: UI5ValidatorsConfig<BindingIssue> =
+    cloneDeep(bindingValidators);
   const issues = validateXMLView({
-    validators: mergeValidators<AnnotationIssue>(
+    validators: mergeValidators([
       actualValidators,
-      externalValidators
-    ),
+      externalFeValidators,
+      externalBindingValidators,
+    ]),
     xmlView: xmlDocAst,
     context: opts.context,
   });
-  const diagnostics = validationIssuesToLspDiagnostics(
+  const diagnostics = validationIssuesToLspDiagnostics<IssueType>(
     issues,
     opts.document,
-    isAnnotationIssue
+    [isAnnotationIssue, isBindingIssue]
   );
   return diagnostics;
 }
 
-function mergeValidators<ExternalIssueType>(
-  v1: UI5ValidatorsConfig<UI5XMLViewIssue>,
-  v2: UI5ValidatorsConfig<ExternalIssueType>
-): UI5ValidatorsConfig<UI5XMLViewIssue | ExternalIssueType> {
-  return {
-    attribute: [...v1.attribute, ...v2.attribute],
-    document: [...v1.document, ...v2.document],
-    element: [...v1.element, ...v2.element],
-  };
+function mergeValidators(
+  param: [
+    UI5ValidatorsConfig<UI5XMLViewIssue>,
+    UI5ValidatorsConfig<AnnotationIssue>,
+    UI5ValidatorsConfig<BindingIssue>
+  ]
+): UI5ValidatorsConfig<IssueType> {
+  return param.reduce(
+    (accumulator: UI5ValidatorsConfig<IssueType>, currentValue) => {
+      return {
+        attribute: [...accumulator.attribute, ...currentValue.attribute],
+        document: [...accumulator.document, ...currentValue.document],
+        element: [...accumulator.element, ...currentValue.element],
+      };
+    },
+    {
+      attribute: [],
+      document: [],
+      element: [],
+    }
+  );
 }
 
-function validationIssuesToLspDiagnostics<
-  ExternalIssueType extends BaseUI5XMLViewIssue & {
-    code?: string | number;
-    tags?: DiagnosticTag[];
-  }
->(
-  issues: (UI5XMLViewIssue | ExternalIssueType)[],
+function validationIssuesToLspDiagnostics<T extends ExternalIssueType>(
+  issues: (UI5XMLViewIssue | T)[],
   document: TextDocument,
-  isExternalIssue: (
-    issue: UI5XMLViewIssue | ExternalIssueType
-  ) => issue is ExternalIssueType
+  externalIssues: ((issue: UI5XMLViewIssue | T) => issue is T)[]
```

Comment:
`externalIssues` is misleading and implies that it is an array of issues, but its actually an array of functions.